library/objective-c/EnvoyEngine.m

```diff
@@ -47,6 +47,14 @@ - (EnvoyHttpStream *)openHttpStreamWithObserver:(EnvoyObserver *)observer {
 
 @end
 
+@implementation EnvoyConfiguration
+
++ (NSString *)templateString {
+    return [[NSString alloc] initWithUTF8String:config_template()];
```

Comment:
Isn't there an init that specifies it won't copy? (I kinda recall that there is?)

Reply:
This could maybe work, but TBH I don't really think copying the bytes here is a big deal since it's a 1-time setup. Also trying this fails to parse when passing it to an Envoy initializer:

```objc
return [[NSString alloc] initWithBytesNoCopy:&config_template
                                      length:strlen(config_template)
                                    encoding:kCFStringEncodingUTF8
                                freeWhenDone:NO];
```